domain-models-runtime/src/main/java/org/folio/rest/security/AES.java

```diff
@@ -26,26 +26,28 @@ public static void setSecretKey(String key) {
   }
 
   /** generate a secret key to use for encrypting a password */
-  public static SecretKey generateSecretKey() throws NoSuchAlgorithmException {
+  /*public static SecretKey generateSecretKey() throws NoSuchAlgorithmException {
     KeyGenerator keyGen = KeyGenerator.getInstance("AES");
     keyGen.init(128);
     SecretKey secretKey = keyGen.generateKey();
     return secretKey;
   }
-
+*/
   /** encrypt a password with the secret key object */
+ /* @SuppressWarnings("all")
   public static byte[] encryptPassword(String password, SecretKey secretKey) throws Exception {
-    Cipher aesCipherForEncryption = Cipher.getInstance("AES");
+    Cipher aesCipherForEncryption = Cipher.getInstance("AES");//using provider-specific default values for the mode and padding scheme.
     aesCipherForEncryption.init(Cipher.ENCRYPT_MODE, secretKey);
     byte[] byteDataToEncrypt = password.getBytes();
     byte[] byteCipherText = aesCipherForEncryption
         .doFinal(byteDataToEncrypt);
     return byteCipherText;
-  }
+  }*/
 
   /** encrypt a password with the secret key and get back a base64 representation of the password */
+  @SuppressWarnings("all")
```

Comment:
Please do not use `@SuppressWarnings("all")`. Instead suppress only one specific warning, for example `@SuppressWarnings("java:S5542")` so that other warnings pop up when we introduce other issues in the code in future.